spec/lib/logging/third_party_transaction_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Logging::ThirdPartyTransaction do
+  # to use shared examples
```

Comment:
(Non blocking) Good call documenting this, might consider more specific language like "To use the shared examples below for the object you are testing" as here it sounds a little like you're documenting RSpec's `shared_examples_for` keyword

Reply:
[PENDING ACTION] will clean up the language

Reply:
done

Reply:
Oh another trick we had at my last gig, in responses like this replying "Done in XXXX" and linking the commit with the change. Again not being nitpicky 🤣 I just thought it was a good idea whoever introduced it there lol